apps/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -91,12 +91,12 @@ add_user_(Admin = #mqtt_admin{username = Username}) ->
 -spec(remove_user(binary()) -> ok | {error, any()}).
 remove_user(Username) when is_binary(Username) ->
     Trans = fun() ->
-                    case lookup_user(Username) of
+                case lookup_user(Username) of
```

Comment:
just realise lookup_user calls dirty_read, maybe create a transactional read API for it.

Reply:
Changed.